src/main/java/com/arpnetworking/metrics/mad/parsers/JsonToRecordParser.java

```diff
@@ -104,6 +106,8 @@ public Record parse(final byte[] data) throws ParsingException {
         final String version = versionNode.get().textValue().toLowerCase(Locale.getDefault());
         try {
             switch (version) {
+                case "2g":
+                    return parseV2gLogLine(jsonNode);
```

Comment:
That is completely up to you.  The reason that we made a steno version was to support shipping the metrics data to a centralized log store at Groupon.  To my knowledge, that never came about.  If you still want to do something like that, make a steno version.   If not, don't.